### DISCUSSION TITLE
Migrating old apiman to apiman 2.0

### DISCUSSION THREAD

**Discussion by sazidknmiet** [apiman/apiman#1369] [Help]
Hello,
I am migrating old apiman to apiman 2.0 so I have following questions:
- What are the names of tables in apiman2 which storing analytics data of api as in previous it was storing in ELK.
- As old apiman data is big so we are unable to migrate from apiman to apiman 2.0 using export import functionality of apiman.
- Due to this, our client have to create new apis and token for using existing apis. **But we don't want to this.**
- **Is there any method client don't have to create new api key and token.**
- Is it  possible to create same api key and token for existing customer in new apiman2 so that client don't have to create new apikey and token.

**Comment by msavy** [comment]
I think you would be best served waiting for Apiman 2.1, which we intend to release this week.

With the new version, issues with the export-import being slow on Elasticsearch have been resolved - even for large setups.

**Reply by sazidknmiet** [reply]
Thanks for your reply. Currently we have apiman 1.5.1.Final and migrating to Apiman 2.0.0.Final.  We have failed many time to export data from Apiman 1.5.1.Final due to slow response from APIMAN server. Also data is around above 10 GB.

Is there any other method to export only selected user data not export all data? ​
If this possible then we will import only selected user data in new apiman.

**Reply by msavy** [reply]
Oh, I see. Hmm, you didn't mention which version you were on. I'm surprised you are facing that issue with 1.5.1.Final. 

Then again, 10GB is the largest Apiman export I have ever heard of by a very large amount; I am impressed! 

It sounds so big I wonder whether that's even plausible, but perhaps you have a very impressive deployment.

If you want, you can reach out to me (look at my profile) and it might be possible for me to see whether there is anything funny going on.
